labeled_data/companies/qingcloud/index.yml

```diff
@@ -9,3 +9,4 @@ data:
     - 86996287 # OpenELB
     - 31385114 # radondb
     - 28995775 # QingCloudAppcenter
+    - 90166053f # quanxiang-cloud
```

Comment:
I think you added an extra `f` in the end of the id

Reply:
Good catch!